libs/openchallenges/ui/src/index.ts

```diff
@@ -4,6 +4,7 @@ export * from './lib/avatar/mock-avatars';
 export * from './lib/challenge-card/challenge-card.component';
 export * from './lib/challenge-card/mock-challenges';
 export * from './lib/challenge-card/mock-platforms';
+export * from './lib/challenge-card/challenge-property-labels';
```

Comment:
Placing the labels in the folder of the `challenge-card` component seems arbitrary. Where could these Records be moved to?

Reply:
yea, I was trying to put it in the `challenge` and `challenge-search` libs, but I got the blank pages. As long as  I call the records is not stored in `ui` lib and the record is called in `challenge-card` component, it will raise error with blank pages. 

 I'm not sure if it's due to some sort of circular dependencies, but I was not able to identify the issue yesterday.

Reply:
What about using a `constants.ts` file like we do in other projects? E.g. in `./lib/constants.ts` (usually data only) and/or `./util.ts` (reusable utility functions)?